### PR TITLE
Documentation improvements and Wiki scripts

### PR DESCRIPTION
**Pull request by milesflo**
Adding a script for generating wiki pages, and another for logging into MongoDB with the library [monk](https://www.npmjs.com/package/monk) and tagging standards

### CODE REVIEW

File: scripts/index.js [hunk 1]
```diff
@@ -20,8 +24,23 @@ let curriculumStats = curriculum.getStats();
...
 // }
 
-for (let topicString in curriculumStats) {
-  let topic = curriculumStats[topicString];
-  console.log(topicString, topic)
+/**
+ * Writes Course pages for wiki in `curriculum.wiki` local directory
+ *
+ */
+function writeCoursePages() {
```
Comment (lizTheDeveloper) [inline scripts/index.js]: index is getting a little weird, this should probably live in an updateWiki.js script in the `/scripts` folder.

File: scripts/standards_scrub.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+      for (let k in course.workouts) {
+        let workout = course.workouts[k];
+        for (insight of workout.insights) {
+          if (insight.standards != undefined) {
+            insightdb.findOne({slug: insight.slug}).then((docs) => {
+              console.log(docs);
+            })
+            for (let s in insight.standards) {
+              let standard = insight.standards[s];
+              // mongoose.insights.find({ "eUID" : insight.slug }).then((data) => {
```
[the remark this answers is not in the document]
Comment (milesflo) [reply scripts/standards_scrub.js]: Ah, this was what was used to tag insights on mongo. In cleaning up, I deleted a comment that had the functioning snippet in it. Will fix or omit, whichever you prefer

File: scripts/index.js (hunk 1 above)
Comment (milesflo) [reply scripts/index.js]: :+1:

File: scripts/standards_scrub.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+const insightdb = db.get("insights");
+
+
+/**
+ * Given a course, iterates down to the standards level for tagging purposes
+ */
+function standardsScrub(course) {
+  // Passed in a parameter...
+  var obj = {};
+  for (let i in curriculum.topics) {
```
[the remark this answers is not in the document]
Comment (milesflo) [reply scripts/standards_scrub.js]: Objects aren't iterable in the same way as arrays.. JS doesn't like using `for..of` on obj keys

File: scripts/update_wiki.js [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+  let data = "";
+
+  // Hard-coded header and table layout.
+  data += `These are the topics that currently exist, with planning and discussion boards for each.\n
+Topic | Subtopics | Workouts | Insights | PQ | RQ | Standards | Assessments | Stubs
+--- | --- | --- | --- | --- | --- | --- | --- | ---
+`;
+
+  // Parse topic obj into table row
+  for (let i in curriculumStats) {
```
[the remark this answers is not in the document]
Comment (milesflo) [reply scripts/update_wiki.js]: :+1:

File: scripts/update_wiki.js [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+
+/**
+ * Updates LOT in local directory.. https://github.com/enkidevs/curriculum/wiki/List-of-Topics
+ *
+ */
+function updateTopicsTable() {
+  let data = "";
+
+  // Hard-coded header and table layout.
+  data += `These are the topics that currently exist, with planning and discussion boards for each.\n
```
[the remark this answers is not in the document]
Comment (milesflo) [reply scripts/update_wiki.js]: :+1:

File: scripts/update_wiki.js [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+
+  // return header + table
+  return data;
+}
+
+
+/**
+ * Generate course-map with clickable links
+ */
+function generateCourseTree() {
```
[the remark this answers is not in the document]
Comment (milesflo) [reply scripts/update_wiki.js]: It is now Course.generateCurriculumTreeMarkdown

File: scripts/update_wiki.js [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+  // Hard-coded header and table layout.
+  data += `These are the topics that currently exist, with planning and discussion boards for each.\n
+Topic | Subtopics | Workouts | Insights | PQ | RQ | Standards | Assessments | Stubs
+--- | --- | --- | --- | --- | --- | --- | --- | ---
+`;
+
+  // Parse topic obj into table row
+  for (let i in curriculumStats) {
+    let topic = curriculumStats[i];
+    data += `[[${toTitleCase(i)} Topic]] | ${topic.courses} | ${topic.workouts} | ${topic.insights} | ${topic.practiceQuestions} | ${topic.revisionQuestions} | ${topic.standards} | ${topic.assessments || 0} | ${topic.stubs}\n`;
```
[the remark this answers is not in the document]
Comment (milesflo) [reply scripts/update_wiki.js]: :+1:

**Comment by milesflo** [comment]
Doing :+1:'s as I go through and make these changes btw

**Pull request closed without merging**